server/src/constants/index.ts

```diff
@@ -0,0 +1,34 @@
+export const AVAILABLE_SUBSCRIPTION_PLANS = [
+  {
+    type: 'superPlan',
+    title: 'Super Plan',
+    priceWithSale: 1100,
+    priceWithoutSale: 1500,
+    duration: '1 month',
+    currency: 'USD',
+  },
+  {
+    type: 'megaPlan',
+    title: 'Mega Plan',
+    priceWithSale: 2200,
+    priceWithoutSale: 3500,
+    duration: '1 month',
+    currency: 'USD',
+  },
+  {
+    type: 'superMegaPlan',
+    title: 'Super Mega Plan',
+    priceWithSale: 3300,
+    priceWithoutSale: 4700,
```

Comment:
this is fine for now of course, although in real life you would never have the field priceWithoutSale, because that would mean you can only have exactly one sale at a time :)